packages/app/components/strategies/Strategies.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import { Tab } from "@headlessui/react";
+
+import { Strategy, useStrategyContext } from "@/contexts";
+import { tabButtonStyles } from "@/app/stacks/stacksOrders";
+
+import { StrategyCard } from "./StrategyCard";
+import { STRATEGY_CATEGORIES } from "./constants";
+
+export const Strategies = () => {
+  const { selectedStrategy, setSelectedStrategy } = useStrategyContext();
+
+  const strategyCategories = Object.values(STRATEGY_CATEGORIES);
+
+  return (
+    <div className="max-w-lg mx-auto mt-8">
+      <Tab.Group>
+        <Tab.List className="mb-2">
+          <div className="flex space-x-2">
+            {strategyCategories.map((category, catIndex) => (
+              <Tab
+                className={tabButtonStyles}
+                key={catIndex}
+                onClick={() => setSelectedStrategy(null)}
+              >
+                {category.label}
+              </Tab>
+            ))}
+          </div>
+        </Tab.List>
+        <Tab.Panels>
+          {strategyCategories.map((category, catIndex) => (
+            <Tab.Panel
+              className="grid sm:grid-cols-1 lg:grid-cols-2 lg:gap-x-4 gap-y-3.5"
+              key={catIndex}
+            >
+              {category.strategies.map((strategy: Strategy, stratIndex) => {
+                const isSelected = selectedStrategy?.id === strategy.id;
+
+                return (
+                  <StrategyCard
+                    key={stratIndex}
+                    onClick={() =>
+                      setSelectedStrategy(isSelected ? null : strategy)
+                    }
```

Comment:
This click is on the card or on the button? Should we move this logic down to the component?

Reply:
Right now is in the button, we can do it over the whole card. Will move the handler to the whole card.

Not sure about moving down the logic into the Card, I'd keep it here and change what triggers it.

Reply:
I gave it a think and I find it better to move the logic to the component. Less props the better IMO.

Reply:
You just need to pass the strategy prop. Everything else can be assessed on the component. And even can remove `useStrategyContext` from here. What do you think?